test/core/base/context.js

```diff
@@ -316,6 +316,12 @@ describe('Context', function() {
 
 		});
 
+		it('should default empty-object include to document', function () {
+			var result = new Context({ include: {}, exclude: {} });
```

Comment:
Yep. Thanks. The empty-object case (though not "supported") is handled by the same logic as the empty array so the test was still passing and I didn't catch it.